scripts/deploy/containerregistry.sh

```diff
@@ -82,6 +82,18 @@ fi
 
 sleep 10
 
+
+if ( echo ${DESTINATION_REGISTRY_HOST} |grep -q icr.io ); then 
```

Comment:
@Amy0511 Rather than specifically check for `icr.io` perhaps it is better to pass through a new boolean property which determines whether we're dealing with signed or unsigned images?